openadapt/record.py

```diff
@@ -24,6 +24,11 @@
 
 from openadapt import config, crud, utils, window
 
+import sounddevice
```

Comment:
@angelala3252 I think we should reorder imports according to the standard we are following. Please see https://github.com/OpenAdaptAI/OpenAdapt/pull/346 :-)